static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -1,66 +1,66 @@
+import type { AdSize } from '@guardian/commercial-core';
+import { adSizes } from '@guardian/commercial-core';
+import { getBreakpoint, getViewport } from 'lib/detect-viewport';
 import config from '../../../lib/config';
-import fastdom from '../../../lib/fastdom-promise';
 import { mediator } from '../../../lib/mediator';
 import { spaceFiller } from '../../common/modules/article/space-filler';
-import { adSizes } from '@guardian/commercial-core';
-import { addSlot } from './dfp/add-slot';
-import { trackAdRender } from './dfp/track-ad-render';
-import { createAdSlot } from './dfp/create-slot';
-
 import { commercialFeatures } from '../../common/modules/commercial/commercial-features';
 import { initCarrot } from './carrot-traffic-driver';
-import { getBreakpoint, getTweakpoint, getViewport } from 'lib/detect-viewport';
+import { addSlot } from './dfp/add-slot';
+import { createAdSlot } from './dfp/create-slot';
+import { trackAdRender } from './dfp/track-ad-render';
 
-const isPaidContent = config.get('page.isPaidContent', false);
+const isPaidContent = config.get<boolean>('page.isPaidContent', false);
 
 const adSlotClassSelectorSizes = {
 	minAbove: 500,
 	minBelow: 500,
 };
 
-const insertAdAtPara = (para, name, type, classes, sizes) => {
+const insertAdAtPara = (
+	para: Node,
+	name: string,
+	type: string,
+	classes?: string,
+	sizes?: Record<string, AdSize[]>,
```

Comment:
Nice! I wonder if this is a good time to export the `SizeMappings` type from somewhere and re-use it here, rather than having to redefine it.

https://github.com/guardian/frontend/blob/9d057a9f38fa0167e59401ed2647b2363ed42af3/static/src/javascripts/projects/commercial/modules/dfp/create-slot.ts#L4

Reply:
Good idea - maybe we export from `create-slot.ts` for now? 
It looks like `Advert.ts` might be a good place to eventually define that type but perhaps in another PR.

Reply:
I'd be happy with that for now. Longer term somewhere in commercial-core probably makes a lot of sense.